scripts/cibuild

```diff
@@ -18,6 +18,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         usage
     else
         ./scripts/update
+        ./scripts/lint
```

Comment:
Not needed, `./scripts/lint` is called from `./scripts/test`: https://github.com/stactools-packages/sentinel1/blob/e27e50bac8050fa5613e282e3350578fc0d35fe7/scripts/test#L20